packages/builder/src/pages/builder/app/[application]/data/datasource/bb_internal/index.svelte

```diff
@@ -2,10 +2,15 @@
   import { Button, Heading, Body, Layout, Modal, Divider } from "@budibase/bbui"
   import CreateTableModal from "components/backend/TableNavigator/modals/CreateTableModal.svelte"
   import ICONS from "components/backend/DatasourceNavigator/icons"
-  import { tables } from "stores/backend"
+  import { tables, datasources } from "stores/backend"
   import { goto } from "@roxi/routify"
 
   let modal
+
+  let internalTablesBySourceId = $tables.list.filter(
```

Comment:
May not be relevant, but this should probably be a reactive statement. I know it won't change at runtime, but I'm just thinking if you were in the data section and did a page reload, there's a chance that this wouldn't update whenever the backend table store initialises. In pratice it might be totally fine to due to other render blocking etc but thought it was worth checking.